channeldb/graph.go

```diff
@@ -153,6 +153,14 @@ var (
 	// case we'll remove all entries from the prune log with a block height
 	// that no longer exists.
 	pruneLogBucket = []byte("prune-log")
+
+	// closedScidBucket is a top-level bucket that stores scids for
+	// channels that we know to be closed. This is used so that we don't
+	// need to perform expensive validation checks if we receive a channel
+	// announcement for the channel again.
+	//
+	// maps: scid -> []byte{}
+	closedScidBucket = []byte("closed-scid")
```

Comment:
Great point, yes currently we cannot rely on the zombieIndex because peers signaling wrong update times in the `reply_channel_range` msg would cause us to delete those old channels from the zombieIndex as pointed out by Eugene:

https://github.com/lightningnetwork/lnd/blob/master/channeldb/graph.go#L2179-L2183